logging/zerolog_adapter/event_builder.go

```diff
@@ -0,0 +1,57 @@
+package logging
+
+import (
+	"fmt"
+
+	"github.com/moira-alert/moira"
+	"github.com/rs/zerolog"
+)
+
+type EventBuilder struct {
+	*zerolog.Event
+}
+
+func (e EventBuilder) Msg(msg string) {
+	if e.Event != nil {
```

Comment:
Do we really need to check for Event in every method? Maybe it can be moved to constructor

Reply:
We don't want panic I think. And what if we add constructor with error in return parameter? Is it will better or maybe it is overhead? These error checks in every method are strange repeats I think, but I can be wrong, maybe it is fine for Moira or for Golang

Reply:
I looked to the source code of zerolog library and discovered that `event == nil` only in case of disabled logs. So we shouldn't treat this as an error and rather just filter `nil` case in all consecutive method calls